changelog

```diff
@@ -1,3 +1,66 @@
+      * Added a death explosion "weapon" to the Incipias ships. (@TheGiraffe3, @Quantumshark)
+      * "Successors: Researchers Out 1" and "Successors: Large Researchers Out 1" will now correctly calculate expected jumps for dynamic payments based on transit by wormhole and Jump Drive. (@tekker2234)
+      * Added display names, instead of using the unique names, for various early Wanderer missions. (@TheGiraffe3)
+      * Gave display names and descriptions to some early Incipias missions. (@TheGiraffe3)
+      * When selling a Jump Drive to the Unfettered, they will no longer provide a replacement Hyperdrive if the Jump Drive was in cargo. (@Hurleveur)
+      * Corrected the date on which "Earth Day Blackout" will offer. (@TheGiraffe3)
+      * The missions "Gegno Anticipation" and "Gegno Intervention" will no longer offer after the other has been offered. (@Saugia)
+      * Added missing passengers to "Remnant: Cognizance 7" and "Remnant: Visit to Paradise 1". (@opusforlife2)
+      * Text in "Remnant: Void Sprites 3" will now correctly account for if the player was given additional information in "Remnant: Void Sprites 1". (@warp-core)
+      * "Remnant: Shattered Light 3" will no longer remain active after "Remnant: Shattered Light 4" has started, preventing the presence of two Shatered Lights at the same time. (@warp-core)
+      * Updated some Korath ship variants to use the "Piercer Missile Launcher" instead of the deprecated "Korath Piercer Launcher". (@Zitchas)
+      * Updated landing messages for some star sprites to improve accuracy. (@tibetiroka)
+    * Engine bugs:
+      * The conversation panel will no longer close immediately when the player dies. (@TomGoodIdea)
+      * Added required precision specifiers to LineShader GLSL code. (@Koranir, @thewierdnut)
+      * Corrected expected syntax description in the error message for incorrect conversation text child nodes. (@warp-core)
+      * Cargo dumped from docked carriable ships is now properly converted into flotsam instead of being lost. (@warp-core)
+      * "Mute" ships, or ships from governments with languages the player does not know will no longer send the "appeasement" message. (@warp-core)
+      * Sold, in stock outfits that are not installable will now correctly be shown as "in stock" instead of "not sold here". (@TomGoodIdea)
+  * Game content:
+    * Mission changes:
+      * Added an "on visit" dialog to, and expanded the descriptions of "Remnant: Cognizance 1". (@warp-core)
+      * Reduced the offer rates of strike breaker jobs by a factor of 5. (@Anarchist2)
+      * Increased the offer rates of culture conversations for single planets in human space. (@reticent-rem)
+    * Graphics:
+      * Added a unique icon for the Swarm Pod, instead of reusing the "meteor" icon. (@Daeridanii1)
+    * Balance:
+      * Reduced the "capture defense" of various outfits with very high values. (@Quantumshark)
+        * Todo: more details.
```

Comment:
```suggestion
        * Knocked down the defense of the Microbot Defense Station from 45 to 18.
        * Cut the defense of the Pug Biodefenses from 250 from 100.
        * Nerfed the Quarg Intrusion Countermeasures from 60 to 24.
```
Or, to keep consistency (though I think the first sounds better):
```suggestion
        * Nerfed the Korath Microbot Defense Station from 45 to 18.
        * Nerfed the Pug Biodefenses from 250 to 100.
        * Nerfed the Quarg Intrusion Countermeasures from 60 to 24.
```

Reply:
These detailed summaries of balance changes are not generally formatted in full sentences like this and I would prefer to kep the brevity of simply giving the name and the old and new values, rather than repeating that it was a "nerf", especially since the introductory line already indicated that it was a reduction.